packages/renderer/src/lib/deployments/deployment-utils.ts

```diff
@@ -29,16 +29,47 @@ export class DeploymentUtils {
     return humanizeDuration(uptimeInMs, { round: true, largest: 1 });
   }
 
+  refreshAge(deployment: DeploymentUI): string {
+    if (!deployment.created) {
+      return '';
+    }
+    // make it human friendly
+    return this.humanizeAge(deployment.created.toString());
+  }
+
   getDeploymentUI(deployment: V1Deployment): DeploymentUI {
+    const conditions = [];
+    if (deployment.status?.conditions) {
+      for (const con of deployment.status.conditions) {
+        const c: DeploymentCondition = {
+          type: con.type,
+          message: con.message,
+        };
+        conditions.push(c);
+      }
+    }
```

Comment:
should be simplified using map like approach or reduce like approach
```suggestion
conditions = (deployment.status.conditions ?? []).map(c => {
          type: con.type,
          message: con.message,
        }
);
```

Reply:
Fixed in new commit.